packages/bun-types/package.json

```diff
@@ -1,25 +1,37 @@
 {
   "name": "bun-types",
```

Comment:
@Electroid @paperdave et all. If we are publishing this can we use the `@bun` scoped package? I notice we own the package name `bun` which I believe allows us to create the scope too. This prevents a bunch of `bun-` stuff.

Reply:
@types/bun essentially aliases to bun-types, so most people won’t actually install it directly.

We could change it, but it’s already so widely used

Reply:
`bun-types` was the default for a long time, while i do think changing to `@bun/types` could be a good idea we would need to have a deprecation notice on `bun-types` and update `@types/bun`